python/lsst/daf/relation/_relation.py

```diff
@@ -248,7 +248,7 @@ def chain(self, rhs: Relation) -> Relation:
         -------
         relation : `Relation`
             New relation with all rows from both relations.  If the engine
-            `preserves order <Engine.preserves_chain_order>` for chains, all
```

Comment:
This method seems to have disappeared so I wasn't sure if the text was still valid.

Reply:
Probably best to just remove the sentence about ordering; IIRC the definition of "preserves order" got subtle due to some edge cases and I realized I didn't actually to preserve ordering here to satisfy use cases, so I'd rather leave it unstated.